sqld/src/utils/services/idle_shutdown.rs

```diff
@@ -37,6 +37,10 @@ impl IdleShutdownLayer {
             watcher: Arc::new(sender),
         }
     }
+
+    pub fn kick(&self) {
```

Comment:
Hmm, `IdleShutdownLayer` contains a `Sender`, it is initialized from a `sender`, and the only thing it does is to pass the `watcher` (i.e. `sender`) to the `IdleShutdownService`, which does `self.watcher.send()`. It surely looks like a sender to me!

Maybe I should add a `IdleKicker` type, that would be isomorphic to the `IdleShutdownLayer`, but would be used to kick the watchdog instead of being used as a Tower layer?

Reply:
Done

Reply:
You're right I read a bit to quickly, this is a sender, but yeah isomorphic type is better, so the layer remains for use in hyper related stuff